## Directions

**1. Download the sample data set or use your own** 

Make sure it's a text file of taxonomy names separated by line.
The file "taxa_names.txt" is an example input file provided in the repo.

**2. Obtain the 16S raw reads from NCBI** 

Using the txt file of sample data retrieve the 16s raw reads from NCBI's public database in FASTA format.

Run this line of code with the txt file to retrieve them:
Run through the python script with the txt file to retrieve them:
```
[code here]
get16sseqs.py
```

**3. Use the generated FASTA file to generate the tree**
**3. Download the ETE toolkit**

Following the instructions and code found in this file:
```
ETE_.txt
```
Will get the ETE enviornment downloaded on to your local machine.
One thing to note is that they recommend using conda and Mac OS to install

**4. Use the generated FASTA file to generate the tree**

Run this line of code with the FASTA file to generate the tree:
- choose a workflow name (-w)
- choose FASTA target seqeunces (-a for amino acids and -n for nucleotides)
@@ -53,3 +66,6 @@ ete3 build -a [fastafile].fasta -w soft_modeltest -o output_tree
```

## Usage

This pipeline has many applications. The Dong and Gao labs at Loyola Universtiy Chicago challenged us to create a pipeline that will take in a list of taxa names, generated by previous metagenomic analysis of organisms, and develop a way to create a tree from them. There are many previously designed software and tools that account for this solution, however our approach exaggerates the need for efficency while using previously made tools.
